s3-zip-store/src/main/scala/zip_partitioner/s3/S3ZipStore.scala

```diff
@@ -0,0 +1,77 @@
+package zip_partitioner.s3
+
+import cats.effect.{IO, Resource}
+import eu.timepit.refined.types.string.NonEmptyString
+import fs2.Pipe
+import fs2.aws.s3.S3
+import fs2.aws.s3.models.Models.{BucketName, FileKey}
+import fs2.compression.{DeflateParams, InflateParams, ZLibParams}
+import io.laserdisc.pure.s3.tagless.Interpreter
+import software.amazon.awssdk.services.s3.S3AsyncClient
+
+import java.util.zip.{ZipEntry, ZipOutputStream}
+
+object S3ZipStore {
+//  private type RIO[R, A] = ReaderT[IO, R, A]
+
+  def transfer(from: NonEmptyString, to: NonEmptyString, fileIdentifier: NonEmptyString, s3AsyncClient: S3AsyncClient): fs2.Stream[IO, Unit] =
```

Comment:
Instead of unit, use the type from the upload call, the user of this function should decide if the discard it